src/simulator/halo_simulator.py

```diff
@@ -210,8 +210,7 @@ def simulated_reach_by_spend(
 
         # convert result to a ReachPoint
         impressions = self._data_set.impressions_by_spend(spends)
-        kplus_reaches = ReachPoint.frequencies_to_kplus_reaches(frequencies)
-        return ReachPoint(impressions, kplus_reaches, spends)
+        return ReachPoint(impressions, frequencies, spends)
```

Comment:
I think we should rename `frequencies` to `kplus_reaches` since `kplus_reaches` is what `ReachPoint` needs.

Reply:
Good point!

Reply:
```python
frequencies = [
            round(x) for x in estimator.estimate_cardinality(combined_sketch)
        ]
```
Could we rename the variable above in line 197 to `kplus_reaches`? I.e.
```python
kplus_reaches = [
            round(x) for x in estimator.estimate_cardinality(combined_sketch)
        ]
```
The thing essentially is `kplus_reaches`. Using `frequencies` is misleading.

Reply:
Good point!